lib/v8.js

```diff
@@ -254,13 +320,25 @@ class DefaultDeserializer extends Deserializer {
   }
 }
 
+/**
+ * Uses a `DefaultSerializer` to serialize `value`
+ * into a buffer.
+ * @param {any} value
+ * @returns {Buffer}
+ */
 function serialize(value) {
   const ser = new DefaultSerializer();
   ser.writeHeader();
   ser.writeValue(value);
   return ser.releaseBuffer();
 }
 
+/**
+ * Uses a `DefaultDeserializer` with default options
+ * to read a JS value from a buffer.
```

Comment:
Might it be better to not abbreviate `JavaScript`? 🤔
```suggestion
 * to read a JavaScript value from a buffer.
```